### PR TITLE
Wait for firewalld polkit policy to be defined

### PR DESCRIPTION
**Pull request by sdodson**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1436964
Fixes https://github.com/openshift/openshift-ansible/issues/3213

### CODE REVIEW

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
[merge]

**Comment by sdodson** [comment]
The CI jobs show that this task was skipped but I guess that's because firewalld was installed and enabled on the host before running these playbooks. Locally I've tested by enabling iptables and disabling firewalld then running the playbooks, things worked fine.

File: roles/os_firewall/tasks/firewall/firewalld.yml [hunk 1]
```diff
@@ -34,6 +34,13 @@
   pause: seconds=10
   when: result | changed
 
+# Fix suspected race between firewalld and polkit BZ1436964
+- name: Wait for polkit action to have been created
+  command: pkaction --action-id=org.fedoraproject.FirewallD1.config.info
+  ignore_errors: true
+  register: pkaction
+  until: pkaction.rc == 0
```
Comment (sdodson) [inline roles/os_firewall/tasks/firewall/firewalld.yml]: Maybe changed_when false?

Comment (mtnbikenc) [inline roles/os_firewall/tasks/firewall/firewalld.yml]: I agree, this should have `changed_when: false`.

**Comment by sdodson** [comment]
Added changed_when: false, retries: 6, delay: 10

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
[merge]

**Pull request merged**